website/docs/advanced/search_path.md

```diff
@@ -3,15 +3,94 @@ id: search_path
 title: Config Search Path
 ---
 
-The Config Search Path is a list of paths that are searched in order to find configs. It is similar to
-the Python PYTHONPATH.
+import {ExampleGithubLink} from "@site/src/components/GithubLink"
 
+The Config Search Path is a list of paths that Hydra searches in order to find configs. It is similar to
+the Python `PYTHONPATH`. 
  - When a config is requested, The first matching config in the search path is used.
- - Each search path element has a schema prefix such as file:// or pkg:// that is corresponding to a ConfigSourcePlugin.
- - `SearchPathPlugin` can manipulate the search path.
+ - Each search path element has a schema prefix such as `file://` or `pkg://` that is corresponding to a `ConfigSourcePlugin`.
 
 You can inspect the search path and the configurations loaded by Hydra by turning on verbose logging for the `hydra` logger:
 
-```text
+```bash
 $ python my_app.py hydra.verbose=hydra
```

Comment:
1. --info config is better.
2. you can add --info searchpath (different pr).
3. the searchpath is incomplete and does not contain the effective searchpath from the config file. You can fix it (another PR).

Reply:
Additionally, I think we should output a warning of a config searchpath is not pointing to a valid location.
You can fix it in a different PR.

Reply:
Created an issue for the follow up needed.